### PR TITLE
doc: update to READMEs

### PR DESCRIPTION
**Pull request by srl295**
- minor update
- split out a `guides/I18N-API`

### CODE REVIEW

**Review by obensource — APPROVED** [review]
@srl295 many thanks for all your updates to these guides, and for contributing this wonderful i18n-api guide. You're the best, and it's very appreciated! 🙏 Looks GREAT to me! 🙌

File: guides/I18N-API.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+You can help by being familiar with these projects, and how changes impact Node.js
+and its users. You can help make sure Node.js is compliant with these specifications.
+These are all projects which invite public participation.
+
+- [ECMA-262](https://tc39.github.io/ecma262/) is the specification for the
+ECMAScript language, and includes specification for international-related
+functionality in the core language.
+
+- [ECMA-402](https://tc39.github.io/ecma402/) is the specification for
+complementary langauge-sensitive functionality in ECMAScript, such as the
```
Comment (JohnTitor) [inline guides/I18N-API.md:41]:
```suggestion
complementary language-sensitive functionality in ECMAScript, such as the
```

File: guides/ROLES.md [hunk 2]
```diff
@@ -1,69 +1,69 @@
...
-Main player in localization process, who actually make translation.
+The main actor in the localization process, as the one which actually performs the translation.
 
-To become a translator, just read our guidelines and go ahead! We really appreciate your volunteering.
+To become a translator, simply read our guidelines and go ahead! We really appreciate your volunteering.
 
 ## GitHub
 
-From GitHub side we have just two type of roles:
+From the GitHub side we have just two type of roles:
```
Comment (JohnTitor) [inline guides/ROLES.md:45]:
```suggestion
From the GitHub side we have just two types of roles:
```

File: guides/ROLES.md [hunk 3]
```diff
@@ -1,69 +1,69 @@
 # Roles
 
-Localization process based on two environments – GitHub and Crowdin. Both of them have roles management, and Node.js i18n group standardized it into one system.
+The Localization process is based on two environments – GitHub and Crowdin. Both of them have role management, and the Node.js i18n working group has standardized it into a unified system.
 
 ## Crowdin
 
-From [official docs](https://support.crowdin.com/modifying-project-participants-roles/#project-roles) of Crowdin, project members hierarchy looks like:
+Per the [official docs](https://support.crowdin.com/modifying-project-participants-roles/#project-roles) of Crowdin, the project has different levels of responsiblities:
```
Comment (JohnTitor) [inline guides/ROLES.md:7]:
```suggestion
Per the [official docs](https://support.crowdin.com/modifying-project-participants-roles/#project-roles) of Crowdin, the project has different levels of responsibilities:
```

**Review by JohnTitor — APPROVED** [review]
Overall looks good to me with some nits.

File: guides/I18N-API.md (hunk 1 above)
Comment (srl295) [reply guides/I18N-API.md]: The one word I can never spell…

**Comment by srl295** [comment]
goal is to unblock #374

**Review by JohnTitor — APPROVED** [review]
Awesome, thanks!

**Review by alexandrtovmach — APPROVED** [review]
LGTM!

**Comment by alexandrtovmach** [comment]
Thank you @srl295, really appreciated fixes and updates :+1:

File: README.md [hunk 4]
```diff
@@ -2,21 +2,12 @@
 
 [![Crowdin](https://badges.crowdin.net/nodejs/localized.svg)](https://crowdin.com/project/nodejs)
 
-Here you can find contribution guides, discuss ideas or join meetings. This repo also contains the source English Node.js API documentation as well as translated versions of the documentation. The source English is copied on a schedule from its canonical home in the [doc](https://github.com/nodejs/node/tree/master/doc) directory of the nodejs/node repository. The translated files are added by pull requests from our Crowdin integration.
+Here you can find contribution guides, discuss ideas, and find out how to join meetings. This repo also contains the source for the English Node.js API documentation as well as translated versions of the documentation. The source English contnet is copied on a schedule from its canonical home in the [doc](https://github.com/nodejs/node/tree/master/doc) directory of the nodejs/node repository. The translated files are added by pull requests from our [Crowdin](./guides/CROWDIN.md) integration.
```
Comment (RichardLitt) [inline README.md:5]: contnet > content

File: README.md [hunk 5]
```diff
@@ -35,12 +26,16 @@ Maintaining the ability for Node.js to [effectively support](https://nodejs.org/
...
 
 - Making the Node.js project's API documentation, site, and tutorial content available in every language.
-- The responsibility to provide translators with a useful platform to work in.
-- The responsibility to provision Node.js with all l10n assets, through coordination with [Node.js](https://github.com/nodejs/node) core contributors.
-- Members of this group are not responsible for performing the act of translation itself.
+- Providing translators with a useful and productive platform to work in.
+- Provisioning Node.js with all l10n assets, through coordination with [Node.js](https://github.com/nodejs/node) core contributors.
+
+Members of this group are not responsible for performing the actual translations,
+but of coordinating the translation among the language-specific l10n groups.
```
Comment (RichardLitt) [inline README.md:38]: but for

**Review by RichardLitt — COMMENTED** [review]
Good work. Some small edits I noticed while skimming through it.

**Pull request merged**